rules-java/api/pom.xml

```diff
@@ -35,6 +35,23 @@
             <version>5.0.4</version>
         </dependency>
 
+        <!-- Local Dependencies -->
+        <dependency>
+            <groupId>org.jboss.windup.maven</groupId>
+            <artifactId>nexus-indexer-data</artifactId>
+            <version>6.0</version>
+            <scope>compile</scope>
+            <optional>true</optional>
+        </dependency>
```

Comment:
so this still stays without definition in windup-bom, this is not external dependency in the reality, right ?